package-res/resources/web/pentaho/data/filter/AbstractFilter.js

```diff
@@ -73,10 +66,7 @@ define([
     },
 
     /**
-     * Returns a filter that is the union of this filter with another.
-     * In other words, implements the OR operation between this filter and another.
-     * @param {} other -
-     * @returns {*}
+     * @inheritdoc
```

Comment:
Running ant-js does not show any docs, because you moved the description to an interface.